### DISCUSSION TITLE
device type library as git data source fails to sync

### DISCUSSION THREAD

**Discussion by misch42** [netbox-community/netbox#14441] [Help Wanted!]
When I add the device type library on github as a git data source the sync fails with a status "errored"

Any ideas?

**Comment by deku-m** [comment]
I was trying the same couldn't get it fixed, following...

**Comment by arcticash** [comment]
I'm having a similar issue with a GitLab repository for config templates, cant seem to work out where the Data Source error is logging to as I cant see anything in the log file and there is no real error message, just the job reporting its errored and the job data is just showing "null"

**Comment by arcticash** [comment]
Just noticed that the logging might be improved in the next release, sounds like #13334 has been included to store the error message along with the job details in the database.
